mirror.go

```diff
@@ -259,32 +271,58 @@ func (m *mirror) getRemoteTags() ([]RepositoryTag, error) {
 		}
 
 		return allTags, nil
-	}
+	} else if m.repo.RemoteTagSource == "registry" {
+		if !strings.Contains(m.repo.Name, "/") {
+			return nil, fmt.Errorf("No registry in image name")
+		}
+		parts := strings.Split(m.repo.Name, "/")
```

Comment:
Should only split in two to handle image namespace. For example `myregistry.io/mynamespace/myimage`.

```
strings.SplitN(m.repo.Name, "/", 2)
```